internal/test/e2e/run.go

```diff
@@ -592,3 +596,10 @@ func logTinkerbellTestHardwareInfo(conf *instanceRunConf, action string) {
 	}
 	conf.Logger.V(1).Info(action+" hardware for TestRunner", "hardwarePool", strings.Join(hardwareInfo, ", "))
 }
+
+func shuffleHardwareInventory(invCatalogue *hardwareCatalogue) {
```

Comment:
nit: why the use of inventory and catalogue? aren't they representing the same thing?